validate.go

```diff
@@ -106,6 +107,29 @@ func checkMounts(mounts []rspec.Mount, rootfs string) {
 	}
 }
 
+func checkHooks(process rspec.Hooks, rootfs string) {
```

Comment:
`process` is a strange name for this variable.  How about `hooks`?  And hooks are in the host mount namespace, so I don't think we need `rootfs`.